index.html

```diff
@@ -7,14 +7,15 @@
     <meta name="robots" content="index, follow">
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link rel="stylesheet" type="text/css" href="src/styles/header.scss">
```

Comment:
By using Vite, you could directly import header.scss directly into header.js

Reply:
could you guide me?

Reply:
Done through a virtual meeting :)